packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import Box from '~components/Box';
+import { useDropdown } from '~components/Dropdown/useDropdown';
+import { Text } from '~components/Typography';
+import { getPlatformType, makeAccessible } from '~utils';
+
+type ActionListItemProps = {
+  title: string;
+  value: string;
+  href?: string;
+  /**
+   * Internally passed from ActionList. No need to pass it explicitly
+   */
+  index?: number;
+};
+const ActionListItem = (props: ActionListItemProps): JSX.Element => {
+  const {
+    activeIndex,
+    dropdownBaseId,
+    onOptionClick,
+    selectedIndices,
+    setShouldIgnoreBlur,
+  } = useDropdown();
+
+  const platformType = getPlatformType();
+  const renderOnWebAs = props.href ? 'a' : 'button';
+  const isReactNative = platformType === 'react-native';
+
+  return (
+    <Box
+      as={!isReactNative ? renderOnWebAs : undefined}
+      id={`${dropdownBaseId}-${props.index}`}
+      role="option"
+      data-value={props.value}
+      data-index={props.index}
+      aria-selected={
+        typeof props.index === 'number' ? selectedIndices.includes(props.index) : undefined
+      }
+      onClick={(e: React.MouseEvent<HTMLButtonElement>) => {
+        if (typeof props.index === 'number') {
+          onOptionClick(e, props.index);
+        }
+      }}
+      onMouseDown={() => {
+        setShouldIgnoreBlur(true);
```

Comment:
I have fixed it upto some level. There is a slight animation sometimes. Will look into it after Dropdown is done. It might need some changes in BaseInput to support ignoring blur completely

Reply:
Yup, the input's bottom border animation is happening on each click <img src="https://user-images.githubusercontent.com/6682655/212875579-b5dc637e-0dc2-46f4-99d9-74b937103c01.png" width="16" />